### ISSUE TITLE
Fresh install; Add my first Hub/Channel. I made a mistake, so I delete it.  App Not Responding..

### ISSUE THREAD

**Issue by Snacko** [iquercorb/OpenModMan#113]
Fresh install
Add my first Hub/Channel. 
I made a mistake, so I delete it.  
OOM Locks up and I have to End Task.

Repeatable...

**Comment by iquercorb** [comment]
Yes, that is because you deleted a file the application may try to access... Logically application should have prevent you to delete file with an access violation error, but due to how XML file are currently handled, it is not the case. 
Fixing this would require major code refactoring, so, simply avoid to do such thing until I found time and energy to rewrite the XML file parsing logic...

**Comment by lmiol** [comment]
when we can try 1.4.0 beta/release?

**Comment by iquercorb** [comment]
> When we can taste 1.4.0 beta/release? 

1.4.0 ? 1.3.1 you mean ? probably in some weeks, as you see there not much bug fix/changes and they are pretty minor...

> Maybe you can create Discord channel?

For three users and a half ?... no.

**Comment by Snacko** [comment]
> When we can taste 1.4.0 beta/release? _i dont like to post here messages like this, because they do not relate to this issue. Maybe you can create Discord channel?_

You can post questions and comments on the DCS forums..

**Comment by lmiol** [comment]
> You can post questions and comments on the DCS forums..

what you talking about?
